core/state_processor.go

```diff
@@ -80,6 +80,10 @@ func (p *StateProcessor) Process(block *types.Block, statedb *state.StateDB, cfg
 	if beaconRoot := block.BeaconRoot(); beaconRoot != nil {
 		ProcessBeaconBlockRoot(*beaconRoot, vmenv, statedb)
 	}
+	if p.config.IsPrague(block.Number(), block.Time()) {
+		// This should not underflow as genesis block is not processed.
+		ProcessParentBlockHash(statedb, vmenv, block.ParentHash())
```

Comment:
I think it would be nicer if the order of argument is similar to `ProcessBeaconBlockRoot`, i.e: `data, vmenv, statedb`). I don't care which order, but they should be similar